.github/workflows/deploy-release.yml

```diff
@@ -18,6 +18,10 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip setuptools wheel
+      - name: Compile localization message files
+        run: |
+          pybabel compile -d mkdocs/themes/mkdocs/locales
+          pybabel compile -d mkdocs/themes/readthedocs/locales
```

Comment:
Hmm, any reason we are not using our own commands here? I'm sure these commands work just fine, but we should be using our own work if we expect our users to use it. Also, that helps to ensure we will notice if/when we introduce any bugs.

Reply:
Absolutely, thanks for spotting it. I missed updating it there while I did update it on the standalone version in setup.py

I tried it out to check that it indeed compiles ALL locales available, and caught a hidden datetime validation bug ;)

I pushed b178419 to switch workflow to use these commands

:+1: good now